pt/lessons/ecto/changesets.md

```diff
@@ -0,0 +1,290 @@
+---
+version: 1.0.0
```

Comment:
I made the review against the 1.1.1 English version, so I believe we can use this version here to keep it in sync.